lib/cisco_node_utils/fabricpath_global.rb

```diff
@@ -231,8 +224,7 @@ def default_linkup_delay_enable
 
     def loadbalance_algorithm
       algo = config_get('fabricpath', 'loadbalance_algorithm')
-      return default_loadbalance_algorithm if algo.nil?
-      algo.first.strip.downcase
+      algo.downcase
```

Comment:
Should be able to just do `config_get('fabricpath', 'loadbalance_algorithm').downcase`